app/graphql/users/middlewares.js

```diff
@@ -0,0 +1,19 @@
+const errors = require('../../errors');
+
+const emailRegex = /^[a-zñ0-9.+_-]+@(wolox.(ar|cl|co|com|com.ar))$/i;
+const passwordRegex = /^[a-zñ0-9._-]{8,}$/i;
+
+exports.createUser = (resolve, root, args) => {
+  const { user } = args;
+  const errorList = [];
```

Comment:
dont do a list of string errors. Do a list of object including message and statuscode